### PR TITLE
Update extreme_ers_ssh.py

### PR DESCRIPTION
**Pull request by shivaraj-arch**
Few Avaya ERS devices after SSH-login the Ctrl-Y turns to blank screen giving no pattern to search; prompt appears after Enter/Return key press ( tested ). With others this is just a Return key press and is safe.

Target Error: ReadTimeout::
Pattern not detected: '(?:sername|ssword|Ctrl-Y|(?m:[>#]\\s*$))' in output.

### CODE REVIEW

**Review by shivaraj-arch — COMMENTED** [review]
require approval from maintainer

**Comment by shivaraj-arch** [comment]
[Netmiko](https://github.com/shivaraj-arch/netmiko/actions/workflows/commit.yaml)

**Comment by ktbyers** [comment]
@shivaraj-arch Can you show me what the SSH login looks like on these Avaya ERS devices?

I.E. copy-and-paste the CLI interaction into this Github issue.

File: .github/workflows/python-app.yml
Comment (ktbyers) [inline .github/workflows/python-app.yml:39]: You should remove this (i.e. this change shouldn't be included in the PR).

**Review by ktbyers — CHANGES_REQUESTED** [review]
You should remove the GitHub actions file, that shouldn't be included in this PR.

**Comment by shivaraj-arch** [comment]
> @shivaraj-arch Can you show me what the SSH login looks like on these Avaya ERS devices?
> 
> I.E. copy-and-paste the CLI interaction into this Github issue.

@ktbyers sorry do not have access to it now, just saw this still lying and hence tried running workflow. Have tested it thoroughly then. It won't affect existing functionality, so no harm in adopting changes.

**Comment by shivaraj-arch** [comment]
> You should remove the GitHub actions file, that shouldn't be included in this PR.

Done.

**Review by ktbyers — APPROVED** [review]

**Comment by ktbyers** [comment]
Superseded by:

https://github.com/ktbyers/netmiko/pull/3324

To fix merge/out-of-date issues.

**Pull request closed without merging**
